src/api/buddies.ts

```diff
@@ -54,11 +78,29 @@ const banRequest = (
   accessToken: authApi.AccessToken,
   status: BanActions,
 ) => {
-  const statusStr = status === 'Ban' ? 'banned' : 'ok';
-
   return http.put(
     `${config.baseUrl}/users/${accessToken.userId}/contacts/${buddyId}`,
-    { status: statusStr },
+    { status: banActionToStatusString(status) },
+    {
+      headers: authApi.authHeader(accessToken),
+    },
+  );
+};
+
+const batchBanRequest = (
+  buddyIds: string[],
+  accessToken: authApi.AccessToken,
+  status: BanActions,
+) => {
+  let buddies = [];
+
+  for (const buddyId of buddyIds) {
+    buddies.push({ id: buddyId, status: banActionToStatusString(status) });
+  }
```

Comment:
Instead of a for loop, this could be introduced with a .map-function

```
const buddies = buddyIds.map(buddyId => { 
    return {
        id: buddyId,
        status: banActionToStatusStrings(status)
    }
})
```
Using maps for mapping lists is just more in point stylewise in this project =)
Also then we dont need to introduce a let variable, and reassign values to it.